tests-integration/ignored_users_test.go

```diff
@@ -161,8 +161,10 @@ func TestIgnoredUsersDuringLiveUpdate(t *testing.T) {
 			},
 		},
 	})
-	t.Log("Alice sees her message.")
-	m.MatchResponse(t, aliceRes, m.MatchRoomSubscription(roomID, m.MatchRoomTimelineMostRecent(1, []json.RawMessage{aliceMsg2})))
+	t.Log("Alice sees her join, her messages and nigel's state in the timeline.")
+	m.MatchResponse(t, aliceRes, m.MatchRoomSubscription(roomID, m.MatchRoomTimeline([]json.RawMessage{
+		aliceJoin, aliceMsg, nigelState, aliceMsg2,
+	})))
```

Comment:
This could probably be an E2E test, to be honest. Sync to get Nigel's messages, then ignore him, then make a new conn. You shouldn't see his messages.

Reply:
Oh, but I guess that will only take effect on a restart, as the events will be kept around in UserRoomData timelines.

Reply:
There is no in-memory cache of timelines anymore.